src/tests/ftest/get_remote_files.sh

```diff
@@ -269,6 +269,11 @@ if ! ret=$(check_files_input "${FILES_TO_PROCESS}"); then
         exit 1
     elif [[ ${ret} -eq 2 ]]; then
         echo "No files matched ${FILES_TO_PROCESS}. Nothing to do."
+        if [ -n "${ARCHIVE_DEST}" ]; then
```

Comment:
This should also take into account whether or not `$VERBOSE` is set as there won't be a log file to copy if it is set.  Double check the bash syntax, but something like this:
```suggestion
        if ! "${VERBOSE}" && [ -n "${ARCHIVE_DEST}" ]; then
```

Reply:
Done